### PR TITLE
N°4765 - Update brazilian translations

### PR DESCRIPTION
**Pull request by eduardomozart**
Removed the ``~~`` flag from translated strings.

### CODE REVIEW

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 1]
```diff
@@ -33,15 +33,15 @@
...
+	'Core:AttributeLinkedSetDuplicatesFound' => 'Duplicatas no campo \'%1$s\' : %2$s',
 
 	'Core:AttributeDashboard' => 'Painel de controle',
 	'Core:AttributeDashboard+' => '',
 
 	'Core:AttributePhoneNumber' => 'Número de telefone',
 	'Core:AttributePhoneNumber+' => '',
 
-	'Core:AttributeObsolescenceDate' => 'Obsolescência data',
+	'Core:AttributeObsolescenceDate' => 'Data de obsolescência ',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Core:AttributeObsolescenceDate' => 'Data de obsolescência',
```

File: dictionaries/pt_br.dictionary.itop.ui.php [hunk 2]
```diff
@@ -368,30 +368,30 @@
...
-	'UI:LoggedAsMessage' => 'Logado como %1$s (%2$s)~~',
-	'UI:LoggedAsMessage+Admin' => 'Logado como %1$s (%2$s, Administrador)~~',
+	'UI:LoggedAsMessage' => 'Logado como %1$s (%2$s)',
+	'UI:LoggedAsMessage+Admin' => 'Logado como %1$s (%2$s, Administrador)',
 	'UI:Button:Logoff' => 'Sair',
 	'UI:Button:GlobalSearch' => 'Pesquisar',
 	'UI:Button:Search' => ' Pesquisar ',
-	'UI:Button:Clear' => ' Clear ~~',
-	'UI:Button:SearchInHierarchy' => 'Search in hierarchy~~',
+	'UI:Button:Clear' => ' Limpar ',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.ui.php:391]:
```suggestion
	'UI:Button:Clear' => ' Limpar',
```

**Review by Molkobain — CHANGES_REQUESTED** [review]
Thanks for the PR! Can you just accept the 2 changes I suggested? (you can click accept directly on the webpage, no need to do another PR)

**Comment by Molkobain** [comment]
Nevermind, I was able to apply them myself.
I'll discuss this PR internally and get back to you soon.

**Review by Molkobain — DISMISSED** [review]

**Comment by piRGoif** [comment]
Hello,
Thanks for your contribution !

I see multiple "+" keys containing same values as their counterparts... For example Class:ActionEmail/Attribute:importance/Value:low and Class:ActionEmail/Attribute:importance/Value:low+.
This is the same in the EN files :/ @Molkobain what do you think ? Shouldn't we fix that ?

**Review by piRGoif — DISMISSED** [review]

**Comment by Hipska** [comment]
> I see multiple "+" keys containing same values as their counterparts... For example Class:ActionEmail/Attribute:importance/Value:low and Class:ActionEmail/Attribute:importance/Value:low+. This is the same in the EN files :/ @Molkobain what do you think ? Shouldn't we fix that ?

I would say, yes!

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 3]
```diff
@@ -254,10 +254,10 @@
...
 	'Class:CMDBChangeOp/Attribute:userinfo' => 'Usuário',
 	'Class:CMDBChangeOp/Attribute:userinfo+' => 'Quem fez essa mudança',
-	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe objeto',
-	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe objeto',
-	'Class:CMDBChangeOp/Attribute:objkey' => 'ID objeto',
-	'Class:CMDBChangeOp/Attribute:objkey+' => 'ID objeto',
+	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objkey' => 'ID do objeto',
+	'Class:CMDBChangeOp/Attribute:objkey+' => 'ID do objeto',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Class:CMDBChangeOp/Attribute:objkey+' => '',
```

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 4]
```diff
@@ -254,10 +254,10 @@
 	'Class:CMDBChangeOp/Attribute:date+' => 'Data e hora da mudança',
 	'Class:CMDBChangeOp/Attribute:userinfo' => 'Usuário',
 	'Class:CMDBChangeOp/Attribute:userinfo+' => 'Quem fez essa mudança',
-	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe objeto',
-	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe objeto',
-	'Class:CMDBChangeOp/Attribute:objkey' => 'ID objeto',
-	'Class:CMDBChangeOp/Attribute:objkey+' => 'ID objeto',
+	'Class:CMDBChangeOp/Attribute:objclass' => 'Classe do objeto',
+	'Class:CMDBChangeOp/Attribute:objclass+' => 'Classe do objeto',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Class:CMDBChangeOp/Attribute:objclass+' => '',
```

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 5]
```diff
@@ -470,10 +470,10 @@
 	'Class:EventLoginUsage+' => 'Conexão com a aplicação',
 	'Class:EventLoginUsage/Attribute:user_id' => 'Login',
 	'Class:EventLoginUsage/Attribute:user_id+' => 'Login',
-	'Class:EventLoginUsage/Attribute:contact_name' => 'Nome usuário',
-	'Class:EventLoginUsage/Attribute:contact_name+' => 'Nome usuário',
-	'Class:EventLoginUsage/Attribute:contact_email' => 'Email usuário',
-	'Class:EventLoginUsage/Attribute:contact_email+' => 'Endereço email deste usuário',
+	'Class:EventLoginUsage/Attribute:contact_name' => 'Nome de usuário',
+	'Class:EventLoginUsage/Attribute:contact_name+' => 'Nome de usuário',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Class:EventLoginUsage/Attribute:contact_name+' => '',
```

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 6]
```diff
@@ -516,40 +516,40 @@
...
+	'Class:ActionEmail/Attribute:body' => 'Corpo',
 	'Class:ActionEmail/Attribute:body+' => 'Conteúdo do email',
-	'Class:ActionEmail/Attribute:importance' => 'importância',
-	'Class:ActionEmail/Attribute:importance+' => 'Flag importância',
-	'Class:ActionEmail/Attribute:importance/Value:low' => 'baixo',
-	'Class:ActionEmail/Attribute:importance/Value:low+' => 'baixo',
+	'Class:ActionEmail/Attribute:importance' => 'Prioridade',
+	'Class:ActionEmail/Attribute:importance+' => 'Flag de prioridade',
+	'Class:ActionEmail/Attribute:importance/Value:low' => 'baixa',
+	'Class:ActionEmail/Attribute:importance/Value:low+' => 'baixa',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Class:ActionEmail/Attribute:importance/Value:low+' => '',
```

File: dictionaries/pt_br.dictionary.itop.core.php [hunk 7]
```diff
@@ -516,40 +516,40 @@
...
 	'Class:ActionEmail/Attribute:importance/Value:normal' => 'normal',
 	'Class:ActionEmail/Attribute:importance/Value:normal+' => 'normal',
-	'Class:ActionEmail/Attribute:importance/Value:high' => 'alto',
-	'Class:ActionEmail/Attribute:importance/Value:high+' => 'alto',
+	'Class:ActionEmail/Attribute:importance/Value:high' => 'alta',
+	'Class:ActionEmail/Attribute:importance/Value:high+' => 'alta',
```
Comment (Molkobain) [inline dictionaries/pt_br.dictionary.itop.core.php]:
```suggestion
	'Class:ActionEmail/Attribute:importance/Value:high+' => '',
```

**Review by Molkobain — DISMISSED** [review]

**Review by Molkobain — DISMISSED** [review]

**Comment by Molkobain** [comment]
@eduardomozart I took the liberty to remove entries for attribute descriptions that were identical to the attribute names. It's a best pratice now but they were indeed already present in the files you translated. :)

**Comment by eduardomozart** [comment]
Hello guys,
I updated some strings from the iTop translation to Portuguese-Brazil during this weekend. Apparently most of the strings were translated with Google Translate at the time, since most strings lacked congruence, especially those that make up the table headers (which justifies this commit having so many translated strings updated). I also adjusted some terms to the Brazilian terms used in the study material and ITIL 4 exam in Brazil.
I intend to continue updating the translation as I use iTop and identify inconsistencies in the translation, as well as the release of new versions of iTop, becoming part of the iTop translation group. I intend to translate modules and extensions as I implement them in my environment.
Thank you for developing iTop, I hope to contribute with bug reports and new features to iTop.

**Comment by Hipska** [comment]
That's great to hear! Currently our (Super-Visions, partner) extensions to iTop are completely translated to Portuguese, but if you would like to sign up, you will be informed if new strings need to be translated: https://www.transifex.com/super-visions/public/

**Comment by eduardomozart** [comment]
The Pull Request "[Fix notifications dropdown menu translation text](https://github.com/Combodo/iTop/pull/268/commits/c4aaabaeea8c9286cb92ec82366c32943e8c3835)" ended up being sent in this same Pull Request, which was not my intention. During the translation work, I found that the text in the "Status" drop-down menu of iTop Notifications was not displayed correctly, instead the translated but generic "Status" of other objects was displayed. By removing the "+" character from the notification parameters, the translated terms started to be displayed correctly. I made the change in the other language files.

**Comment by Hipska** [comment]
Please mark this PR as draft asl long as you are still working on it.

**Comment by piRGoif** [comment]
Hello,

> Please mark this PR as draft asl long as you are still working on it.

Yes please :)
And when you are ready for us to review your pr, add a comment stating so.

Thanks !

File: datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php [hunk 8]
```diff
@@ -33,7 +33,7 @@
 	'Attachment:Max_Go' => '(Tamanho máximo de arquivo: %1$s Gb)',
 	'Attachment:Max_Mo' => '(Tamanho máximo de arquivo: %1$s Mb)',
 	'Attachment:Max_Ko' => '(Tamanho máximo de arquivo: %1$s Kb)',
-	'Attachments:NoAttachment' => 'Nenhum anexo. ',
+	'Attachments:NoAttachment' => 'Nenhum anexo ',
```
Comment (Molkobain) [inline datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php]:
```suggestion
	'Attachments:NoAttachment' => 'Nenhum anexo',
```

**Review by Molkobain — CHANGES_REQUESTED** [review]

**Comment by piRGoif** [comment]
> The Pull Request "[Fix notifications dropdown menu translation text](https://github.com/Combodo/iTop/pull/268/commits/c4aaabaeea8c9286cb92ec82366c32943e8c3835)" ended up being sent in this same Pull Request, which was not my intention. During the translation work, I found that the text in the "Status" drop-down menu of iTop Notifications was not displayed correctly, instead the translated but generic "Status" of other objects was displayed. By removing the "+" character from the notification parameters, the translated terms started to be displayed correctly. I made the change in the other language files.

By "itop notification" do you mean the bell icon on the bottom left of the UI ? 
Can you share some screenshots of the wrong behavior ?

**Comment by eduardomozart** [comment]
Hello @piRGoif 
The requested screenshots are in attachment:

Before the fix:

before-fix

After the fix:

after-fix
.

**Comment by Molkobain** [comment]
Hello Eduardo,

Product team is ok to include the translations in the future 3.0.2 of iTop. Do you think you will have finished what you want to translate by then? (After summer)

**Comment by eduardomozart** [comment]
Hello,
Sorry for the late reply. Yes, the language file is near from complete and can be merged. Please close this pull request so I can pull the repo again and update newer strings added between 3.0.1/3.0.2 release.

**Comment by eduardomozart** [comment]
Hello.

Why this translation was considered invalid? There was something missing?

**Comment by piRGoif** [comment]
Hello,
You closed this PR on may 9 ? You said :

> Please close this pull request so I can pull the repo again and update newer strings added between 3.0.1/3.0.2 release.

The "invalid" label for us means precisely that : the PR cannot be merged and will be replaced by something else.

Did you create a new PR ? Do you wish to restart work on this one ?

**Comment by eduardomozart** [comment]
Sorry, I closed this to say it was finished, not to end it as it is. I fixed the inconsistences and now it can be safely merged. Tested this translation at our production environment and it seems to be working just fine. Hope to see this translation into next iTop major version, as it will greatly improve PT-BR experience for Portuguese users. We can continue using this PR. Thank you!

**Comment by piRGoif** [comment]
Oh ok ! Close is a final state, opposed to merge (a close pr is unmerged)
Thanks for updating your fork branch !
We will review your modifications !

File: datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php (hunk 8 above)
Comment (Molkobain) [reply datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php]: @eduardomozart please accept this change so we can merge your PR.
You can also check the "allow maintainers to edit PR" on the PR so we can apply changes directly :)

Comment (eduardomozart) [reply datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php]: @Molkobain this merge was already applied to the PR and "Allow edits by maintainers" was already enable for this PR. Please let me know if anymore changes are needed.

Comment (Molkobain) [reply datamodels/2.x/itop-attachments/dictionaries/pt_br.dict.itop-attachments.php]: Indeed it was fixed in the meantime, but was not reflected on my GitHub GUI. Thanks!

**Review by Molkobain — APPROVED** [review]

**Comment by Molkobain** [comment]
Accepted in functional review, will be part of the upcoming 3.1.0

**Comment by Molkobain** [comment]
Hello @eduardomozart , translations will part of the upcoming 3.1.0; thanks again for the contribution! 🙌
Could you send me your fullname and postal address so we can send you contributor stickers?

Cheers,
Guillaume

**Comment by eduardomozart** [comment]
Hello @Molkobain, thank you! Of course I can. Here's my address information (but there's any problem if you can't send it to my country):

Full Name: Eduardo Mozart de Oliveira
Address: ******
City: *******
State: ***
Zip Code: ******
Country: ********

I pretend to keep contributing with iTop with translations and backport some GLPI plugins (like ["accounts"](https://github.com/InfotelGLPI/accounts) plugin) as an iTop extension and offer an iTop course at Udemy at the near future.

_Edited by Molkobain to obfuscate personal address_

**Comment by Molkobain** [comment]
Thanks! I'll send them this week, keep up the good work! 🤩🤩🤩

**Pull request merged**